src/parser/ast.c

```diff
@@ -252,6 +254,17 @@ AST_Validation _Validate_WHERE_Clause(const AST_Query *ast, char **reason) {
   return res;
 }
 
+AST_Validation _Validate_SKIP_Clause(const AST_Query *ast, char **reason) {
+  if(!ast->skipNode) return AST_VALID;
+
+  if(ast->skipNode->skip < 0) {
+    asprintf(reason, "Invalid input '%lld' is not a valid value, must be a positive integer", ast->skipNode->skip);
```

Comment:
An `int64_t` is just an `ld` on my platform - is yours different?

Reply:
changed to size_t, as a result this validation is not necessary.